app/graphql/types/query_type.rb

```diff
@@ -61,18 +62,19 @@ def record_id(id:, index:)
 
         # applied facets
         argument :content_type_facet, [String], required: false, default_value: nil,
-                                                description: 'Filter results by content type'
+                                                description: 'Filter results by content type. Use the `contentType` aggregation of a list of possible values'
```

Comment:
I like these additions, but this set I think we have to many `of`s and not enough `for`s? (I think either `of` could be replaced with a `for` maybe? Applies to all in this class of description.

Reply:
Serves me right for pushing code after 5pm. 🤦